packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import './Card.scss';
+
+const propTypes = {
+  /**
+   * Provides 10px on top and bottom
```

Comment:
Is this padding going to be themeable? If so, you should probably reword this to "Provide themeable padding to top and bottom" same with vertical padding.